flipperdevices/flipperzero-good-faps
# Update MFKey to v3.0

# What's new

This pull request only serves to track changes (as the source may be relocated for OFW 1.1).

- Added Static Encrypted Nested key recovery
- Added NFC app support
- Dropped FlipperNested support

# Verification

Using PR 3822: https://github.com/flipperdevices/flipperzero-firmware/pull/3822

- Acquire weak PRNG MIFARE Classic tag with unknown keys.
- Open NFC app.
- Select "Read" and hold the Flipper Zero against the tag. Wait until tag is fully read.
- Open MFKey manually/select "Recover keys" in NFC app.
- Press center button on Flipper to initiate key recovery. **New keys must be found in this step.**
- Return to NFC app and select "Read". Wait until tag is fully read.
- **Confirm all keys are found on tag.**

# Checklist (For Reviewer)
